cx-server-companion/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:11-alpine
+
+ARG npm_registry=https://registry.npmjs.org/
+ARG cx_server_base_uri=https://raw.githubusercontent.com/SAP/devops-docker-images/master/jenkins-master/REPLACE-ME
+
+#Install dependencies for running the cx server script in this container
+RUN apk add --no-cache bash docker curl && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+WORKDIR /cx-server
+
+COPY files/* ./
+COPY life-cycle-scripts/* ./life-cycle-scripts/
+# Package cx-server(.bat) in this image so we can update them for the user without curl-ing
+ADD ${cx_server_base_uri} ${cx_server_base_uri}.bat ./
+
+RUN npm config set registry=$npm_registry&& \
+    npm install && \
+    npm config delete registry && \
+    # If the repository was cloned on Windows, the script might have \CR\LF line endings. Ensure it has only \LF.
```

Comment:
As far as I understand it would be possible to address this issue by `core.autocrlf` config property. In case I have the right understanding of that issue I would like to recommand to fix this this way.

Reply:
I took a look at `core.autocrlf` config. I think using `dos2unix` avoids putting configuration responsibility on the user's shoulder. Though I do not have strong reservation against writing a few lines of documentation, any reduction of configuration overhead is always good for easy adoption. This is relevant only when the user wants to build the `companion` image locally and the user is using `Windows` OS.